src/components/Account/AccountContextMenu.tsx

```diff
@@ -72,6 +73,8 @@ interface MenuProps {
 function AccountContextMenu(props: MenuProps) {
   const accountData = useLiveAccountData(props.account.publicKey, props.account.testnet)
   const isSmallScreen = useIsMobile()
+  const { t } = useTranslation()
```

Comment:
Since we are only gonna use messages on the `account-context-menu` namespace, let's [scope `t` to this namespace](https://react.i18next.com/latest/usetranslation-hook#loading-namespaces):

```ts
  const { t } = useTranslation("account-context-menu")
```

Reply:
(`page.account.context-menu` or similar if we go by the naming scheme I propose below)